easybuild/easyblocks/o/openssl_wrapper.py

```diff
@@ -30,7 +30,11 @@
 import os
 import re
 
-from urllib.parse import urlparse
+# support python2
+try:
+    from urllib.parse import urlparse
+except ImportError:
+    from urlparse import urlparse
```

Comment:
it's a bit unclear which one retains support for Python 2.x, so let's change to:

```suggestion
try:
    from urllib.parse import urlparse
except ImportError:
    # fallback for Python 2.7, should be removed for EasyBuild >= 5.0
    from urlparse import urlparse